src/GregClientSandbox/Program.cs

```diff
@@ -111,8 +113,19 @@ private static void GetWhitelistTest()
             var nv = WhitelistHeaderCollectionDownload.All();
             var pkgResponse = pmc.ExecuteAndDeserializeWithContent<List<PackageHeader>>(nv);
 
+            if (pkgResponse == null)
+            {
+                Console.WriteLine("The package response is null.");
+                return;
+            }
+
             Console.WriteLine(pkgResponse.message);
-            Console.WriteLine(pkgResponse.content);
+
+            var libs = pkgResponse.content.SelectMany(p => p.versions.Last().node_libraries).Distinct();
```

Comment:
This was updated to show a distinct list of assembly names from only the latest versions.